_data/payments.yml

```diff
@@ -186,9 +186,10 @@ websites:
     url: https://skrill.com/
     img: skrill.png
     tfa:
-      - totp
-      - hardware
-    doc: https://www.skrill.com/en/support#/path/ACCOUNT/Two-Factor-Authentication/905761532
+      - email
+      - sms
+    doc: https://www.skrill.com/en/support#/path/SECURITY/1362150272/
+    exception: "Email method requires an additional PIN"
```

Comment:
What does additional PIN mean?

Reply:
When you setup 2FA, you must create a PIN that then will be used when trying to authenticate using Email. TOTP sent by Email + PIN is one of the options, TOTP sent by SMS (no PIN required) is the other.